src/cpp/rtps/builtin/data/ParticipantProxyData.cpp

```diff
@@ -25,14 +25,14 @@
 #include <fastdds/rtps/builtin/data/WriterProxyData.h>
 #include <fastdds/rtps/builtin/data/ReaderProxyData.h>
 #include <fastdds/rtps/builtin/discovery/participant/PDPSimple.h>
-#include <rtps/network/NetworkFactory.h>
 #include <fastdds/rtps/resources/TimedEvent.h>
 #include <fastrtps/utils/TimeConversion.h>
 
 #include <fastdds/core/policy/ParameterList.hpp>
 #include <fastdds/core/policy/QosPoliciesSerializer.hpp>
 #include <fastrtps_deprecated/participant/ParticipantImpl.h>
```

Comment:
This header file does not seem to be used (and being related to the deprecated API I suggest taking the opportunity and removing it).
Please sort the includes with the recommended order:

* Corresponding header file (ParticipantProxyData.h)
* Blank line
* Standard C header files (there is usually a C++ header file that is preferred so no C header files is advisable).
* Blank line
* Standard C++ files 
* Blank line
* External libraries header files
* Blank line
* Public header files from this library
* Blank line
* Private header files

Each level shall be sorted alphabetically.

This style is not followed in a lot of files, but whenever a header is included or changed, we try to apply this order and try unifying the library code style.

Please, apply in the other files affected by this PR.